source/module_base/global_function.h

```diff
@@ -153,11 +153,11 @@ static void READ_VALUE(std::ifstream &ifs, T &v)
     return;
 }
 
-bool SCAN_BEGIN(std::ifstream &ifs, const std::string &TargetName, const bool restart=1);
+bool SCAN_BEGIN(std::ifstream &ifs, const std::string &TargetName, const bool restart=1, const bool ifwarn=true);
 // Mohan warning : the last term can't be written as const bool &restart,
 // I don't know why.
```

Comment:
Please add comment to modify this annotation. 
When common function is changed, annotation should be in front of it to describe what it is supposed to do.